opensearch/src/main/java/org/opensearch/sql/opensearch/request/OpenSearchScrollRequest.java

```diff
@@ -34,7 +33,7 @@
 public class OpenSearchScrollRequest implements OpenSearchRequest {
 
   /** Default scroll context timeout in minutes. */
-  public static final TimeValue DEFAULT_SCROLL_TIMEOUT = TimeValue.timeValueMinutes(1L);
+  public static final TimeValue DEFAULT_SCROLL_TIMEOUT = TimeValue.timeValueMinutes(100L);
```

Comment:
https://github.com/Bit-Quill/opensearch-project-sql/pull/226#discussion_r1145481302
Use configurable timeout: `plugins.sql.cursor.keep_alive`

Reply:
I have a fix for that, but I found that OpenSearch ignores this parameter.
Tested with values < 1m, scroll context isn't expired even after 2m.
Investigating, WIP.

Reply:
Fixed in 4948ac443.